.github/workflows/render-site.yaml

```diff
@@ -0,0 +1,49 @@
+name: Check rendered site for broken links
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Quarto
+      run: |
+        wget https://github.com/quarto-dev/quarto-cli/releases/download/v1.5.53/quarto-1.5.53-linux-amd64.deb
+        sudo dpkg -i quarto-1.5.53-linux-amd64.deb
+# idk about this
+
+    - name: Install dependencies
+      run: |
+        if [ -f _extensions/quarto-ext.yml ]; then
+          quarto install extension
+        fi
+# idk about this one too
```

Comment:
I think this is OK but I'm not convinced it does anything, as we don't seem to have that file in our folder structure. The extensions we need are all already installed and would be included when the repo gets cloned as part of the CI check.